sway/handlers.c

```diff
@@ -220,6 +220,10 @@ static bool handle_view_created(wlc_handle handle) {
 			// refocus in-between command lists
 			set_focused_container(newview);
```

Comment:
I get a short flash of the new focused window, before the fullscreened regain focus. I wonder if it can be handled here, but it might have some other side effects.